### PR TITLE
ExPlat: Add one-off A/A

### PR DESCRIPTION
**Pull request by jessie-ross**
#### Changes proposed in this Pull Request

* Adds an A/A experiment

See 612-gh-Automattic/experimentation-platform

#### Testing instructions

* N/A

### CODE REVIEW

File: client/signup/steps/domain-upsell/index.jsx [hunk 1]
```diff
@@ -50,6 +50,8 @@ export default function DomainUpsellStep( props ) {
 	const [ isLoadingExperimentAssignment, experimentAssignment ] = useExperiment(
 		'domain_upsell_emphasize_free_v3'
 	);
+	// An A/A experiment to check for anomalies in this location:
+	useExperiment( 'explat_test_aa_one_off_20210421' );
```
Comment (yanirs) [inline client/signup/steps/domain-upsell/index.jsx:54]: A better name may be `explat_test_aa_domain_upsell_20210421`

**Review by yanirs — APPROVED** [review]
Not much to review here :slightly_smiling_face: 
I think it's worth giving the experiment a more informative name before merging.

**Comment by jessie-ross** [comment]
> I think it's worth giving the experiment a more informative name before merging.

I think it's fine as there is a comment explaining it and it is only temporary

**Comment by yanirs** [comment]
> I think it's fine as there is a comment explaining it and it is only temporary

:disappointed:
Our experiment names are forever.

**Comment by jessie-ross** [comment]
We have so many test experiment names... It is linked to an issue and isn't named all that badly.

**Comment by yanirs** [comment]
I don't think it's a good reason to not choose better names when we can, as the name is often used without context.

Anyway, next time I'll just make it explicit that I think it's important by not approving the PR. :slightly_smiling_face:

**Pull request merged**